src/modules/auth/jwt/jwt.strategy.ts

```
@@ -0,0 +1,32 @@
+
+    async validate(payload: {email: string}){
+        let user: any = {};
+
+        user = await this.adminService.getUserByEmail(payload.email);
+        
+        if(!user){
+            throw new UnauthorizedException("User not found, or not authorized.");
+        }
+
+        if (user) {
+            delete user.password;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)